ui/admin/app/templates/scopes/scope/targets/target/workers.hbs

```diff
@@ -51,9 +51,13 @@
                   'resources.target.workers.accordion-label.ingress-workers'
                 }}</span>
               <Hds::Link::Standalone
-                @icon='edit'
+                @icon={{if @model.ingress_worker_filter 'edit' 'plus-circle'}}
```

Comment:
nit: there's text right above this dropdown that feels kind of like it's floating between two sections, rather than helper text that sits on top of the ingress/egress accordions. Wdyt?
<img width="1612" alt="Screenshot 2024-12-09 at 11 44 49 AM" src="https://github.com/user-attachments/assets/f58a453b-4471-435f-b4d6-a6a16a99fe1e">

Reply:
I see what you mean. We should reach out to design and see what they think about this.

Reply:
Not sure I am understanding, but this text, is part of the worker diagram component. Isn't this intendeed?

Reply:
Talked with design and we will keep this as is for now.